ctapipe/image/charge_extractors.py

```diff
@@ -2,18 +2,60 @@
 Charge extraction algorithms to reduce the image to one value per pixel
 """
 
-__all__ = ['ChargeExtractor', 'FullIntegrator', 'SimpleIntegrator',
-           'GlobalPeakIntegrator', 'LocalPeakIntegrator',
-           'NeighbourPeakIntegrator', 'AverageWfPeakIntegrator']
+__all__ = [
+    'ChargeExtractor',
+    'FullIntegrator',
+    'SimpleIntegrator',
+    'GlobalPeakIntegrator',
+    'LocalPeakIntegrator',
+    'NeighbourPeakIntegrator',
+    'AverageWfPeakIntegrator',
+]
 
 
 from abc import abstractmethod
 import numpy as np
-from traitlets import Int, Float
+from traitlets import Int
 from ctapipe.core import Component
 from ctapipe.utils.neighbour_sum import get_sum_array
 
 
+def extract_charge_from_peakpos_array(waveforms, peakpos, width, shift):
+    """
+    Build the numpy array of bools defining the integration window.
+
+    Parameters
+    ----------
+    waveforms : ndarray
+        Waveforms stored in a numpy array.
+        Shape: (n_chan, n_pix, n_samples)
+    peakpos : ndarray
+        Numpy array of the peak position for each pixel.
+        Shape: (n_chan, n_pix)
+    width : ndarray or int
+        Window size of integration window.
+        Shape (if numpy array): (n_chan, n_pix)
+    shift : ndarray or int
+        Window size of integration window.
+        Shape (if numpy array): (n_chan, n_pix)
+
+    Returns
+    -------
+    charge : ndarray
+        Extracted charge.
+        Shape: (n_chan, n_pix)
```

Comment:
The docu regarding Returns is not correct: it returns charge **and** integration window

Reply:
Fixed